File: csharp-dotnet2/src/main/CsharpDotNet2/traveltimeplatform/Model/ResponseError.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace traveltimeplatform.Model {

  /// <summary>
  /// 
  /// </summary>
  [DataContract]
  public class ResponseError {
    /// <summary>
    /// Gets or Sets HttpStatus
    /// </summary>
    [DataMember(Name="http_status", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "http_status")]
    public int? HttpStatus { get; set; }

    /// <summary>
    /// Gets or Sets ErrorCode
    /// </summary>
    [DataMember(Name="error_code", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "error_code")]
    public int? ErrorCode { get; set; }

    /// <summary>
    /// Gets or Sets Description
    /// </summary>
    [DataMember(Name="description", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "description")]
    public string Description { get; set; }

    /// <summary>
    /// Gets or Sets DocumentationLink
    /// </summary>
    [DataMember(Name="documentation_link", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "documentation_link")]
    public string DocumentationLink { get; set; }

    /// <summary>
    /// Gets or Sets AdditionalInfo
    /// </summary>
    [DataMember(Name="additional_info", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "additional_info")]
    public Dictionary<string, List<string>> AdditionalInfo { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class ResponseError {\n");
      sb.Append("  HttpStatus: ").Append(HttpStatus).Append("\n");
      sb.Append("  ErrorCode: ").Append(ErrorCode).Append("\n");
      sb.Append("  Description: ").Append(Description).Append("\n");
      sb.Append("  DocumentationLink: ").Append(DocumentationLink).Append("\n");
      sb.Append("  AdditionalInfo: ").Append(AdditionalInfo).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}

File: dart-jaguar/lib/model/response_error.dart
import 'package:jaguar_serializer/jaguar_serializer.dart';


part 'response_error.jser.dart';

class ResponseError {
  
  @Alias('http_status', isNullable: false,  )
  final int httpStatus;
  
  @Alias('error_code', isNullable: false,  )
  final int errorCode;
  
  @Alias('description', isNullable: false,  )
  final String description;
  
  @Alias('documentation_link', isNullable: false,  )
  final String documentationLink;
  
  @Alias('additional_info', isNullable: false,  )
  final Map<String, List<String>> additionalInfo;
  

  ResponseError(
      

{
     this.httpStatus = null,  
     this.errorCode = null,  
     this.description = null,  
     this.documentationLink = null,  
     this.additionalInfo = const {} 
    
    }
  );

  @override
  String toString() {
    return 'ResponseError[httpStatus=$httpStatus, errorCode=$errorCode, description=$description, documentationLink=$documentationLink, additionalInfo=$additionalInfo, ]';
  }
}

@GenSerializer(nullableFields: true)
class ResponseErrorSerializer extends Serializer<ResponseError> with _$ResponseErrorSerializer {

}

